mne/io/eeglab/eeglab.py

```diff
@@ -596,6 +630,28 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
         logger.info('Ready.')
 
 
+def _check_boundary(annot, event_id):
+    if event_id is None:
+        event_id = dict()
+    if "boundary" in annot.description and "boundary" not in event_id:
+        warn("The data contains 'boundary' events, indicating data "
+             "discontinuities. Be cautious of filtering and epoching around "
+             "these events.")
+
+
+def _check_latencies(latencies):
+    if (latencies < -1).any():
+        raise ValueError('At least one event sample index is negative. Please'
```

Comment:
this is not cover. Does anyone have an example?

Reply:
I had such file somewhere but it might be easier to modify a file, save and load in the test itself (like some eeglab tests do).

Reply:
@mmagnuski should have added a test in the past... :(

we have a lot of eeglab tests without new extra file. Can you propose one?

Reply:
@agramfort - actually there was a test file for this originally: https://github.com/mne-tools/mne-python/pull/4951/files
You think we should add a separate file for this test? I don't think I have any - this should not happen in general, but we have it as a safety check.

Reply:
a better link to the past: https://github.com/mne-tools/mne-python/pull/4951/files#diff-17ae6259472b040a1441c4d11af2bc06R143

Reply:
and more detective work uncovers that @agramfort was the one who didn't want the test in jona's PR: https://github.com/mne-tools/mne-python/pull/5407#issuecomment-411739795
😄